src/rgw/CMakeLists.txt

```diff
@@ -189,6 +194,10 @@ add_dependencies(radosgw cls_rgw cls_lock cls_refcount
   cls_version cls_replica_log cls_user)
 install(TARGETS radosgw DESTINATION bin)
 
+if (WITH_RADOSGW_OPENSSL)
+  target_link_libraries(radosgw ${OPENSSL_LIBRARIES})
```

Comment:
@mdw-at-linuxbox recently argued against linking directly to openssl libs (in discussions about ssl for the beast frontend). Marcus, do you think we should use `dlopen(LIBCRYPTO_SONAME)` for this stuff instead? is there any risk that civetweb and libcurl would be linking this differently?